shared/src/main/scala/amf/core/client/scala/AMFGraphBaseUnitClient.scala

```diff
@@ -56,64 +57,69 @@ class AMFGraphBaseUnitClient private[amf] (protected val configuration: AMFGraph
 
   /**
     * Transforms a [[BaseUnit]] with the default configuration
-    * @param bu [[BaseUnit]] to transform
+    * @param baseUnit [[BaseUnit]] to transform
     * @return An [[AMFResult]] with the transformed BaseUnit and it's report
     */
-  def transform(bu: BaseUnit): AMFResult = AMFTransformer.transform(bu, configuration) // clone? BaseUnit.resolved
+  def transform(baseUnit: BaseUnit): AMFResult =
+    AMFTransformer.transform(baseUnit, configuration) // clone? BaseUnit.resolved
 
   /**
     * Transforms a [[BaseUnit]] with a specific pipeline
-    * @param bu [[BaseUnit]] to transform
+    * @param baseUnit [[BaseUnit]] to transform
     * @param pipelineName name of any custom or [[AMFGraphConfiguration.predefined predefined]] pipeline
     * @return An [[AMFResult]] with the transformed BaseUnit and it's report
     */
-  def transform(bu: BaseUnit, pipelineName: String): AMFResult =
-    AMFTransformer.transform(bu, pipelineName, configuration) // clone? BaseUnit.resolved
+  def transform(baseUnit: BaseUnit, pipelineName: String): AMFResult =
+    AMFTransformer.transform(baseUnit, pipelineName, configuration) // clone? BaseUnit.resolved
+
+  def transformWithPipelineId(baseUnit: BaseUnit, pipelineId: String): AMFResult =
```

Comment:
missing implementation in client.platform

Reply:
fixed